instance/config.py.sample.dev

```diff
@@ -15,11 +15,12 @@ POSTCODE_TIER_OVERRIDE = """{"TE22RR" : {"tier": 2, "shielding": 0},
 # NHS OIDC config
 NHS_OIDC_AUTHORITY_URL = "https://auth.sandpit.signin.nhs.uk"
 NHS_OIDC_CLIENT_ID = "temp"
-NHS_OIDC_REGISTRATION_CALLBACK_URL = "http://localhost:5000/nhs-registration-callback"
-NHS_OIDC_LOGIN_CALLBACK_URL = "http://localhost:5000/nhs-login-callback"
-NHS_OIDC_LOGIN_PRIVATE_KEY_PATH = "instance/private_key.pem"
+NHS_OIDC_REGISTRATION_CALLBACK_URL = "http://webform:5000/nhs-registration-callback"
+NHS_OIDC_LOGIN_CALLBACK_URL = "http://webform:5000/nhs-login-callback"
+NHS_OIDC_SCOPES = ["openid", "profile", "email", "phone", "profile_extended"]
+NHS_OIDC_VTR = ["P0.Cp.Cd", "P0.Cp.Ck", "P0.Cm"]
```

Comment:
The NHS_OIDC values are talked about here https://digital.nhs.uk/services/nhs-care-identity-service-2/guidance-for-developers/openid-connect-overview

The scope is the data we expect back from the NHS. The VTR is the vector of trust. https://nhsconnect.github.io/nhslogin/vectors-of-trust/ P0 is proof of email and phone number.